src/components/ChatView.tsx

```diff
@@ -102,8 +104,15 @@ export const ChatView = ({
     }
   }, []);
 
+  const isInIframe = window !== window.parent;
+
+  const showNavBar = !isInIframe && FEAT_UPDATED_DESIGN;
+
   return (
-    <div id={styles.chatview} data-testid="chatview">
+    <div
+      id={showNavBar ? styles.updatedChatView : styles.chatview}
```

Comment:
For the updated version, we don't need the chat to be 100% view height because that will push the header out of view.